js/utils.js

```diff
@@ -128,6 +128,10 @@ function getRandomShade(colour) {
   return shade+'88'; //Semitransparency
 }
 
+function capitalize(str) {
+  return str.charAt(0).toUpperCase()+str.slice(1);
```

Comment:
Since you do this nicely, how about making sure that only the first letter will be uppercase, and all the rest lowercase?

`str.charAt(0).toUpperCase()+str.slice(1).toLowerCase();`

This would prevents `CamelCase` or `Title Case` roles.

Reply:
I'm not sure that's something we want to prevent, honestly. In particular, if there are e.g. roles that are all-uppercase (acronyms, say ABC) I wouldn't want to flatten that into Abc just for the sake of it.